other/edge-server/client/README.md

```diff
@@ -9,15 +9,16 @@ React client -> Express server (Node.js Driver) -> Edge Server -> Atlas
 
 To use this example application, you must:
 
-1. Create an App Services App based on the `edge-server.todo` template app, and
-   get Edge Server enabled for your new app.
-2. Download, configure, and start the Edge Server.
-3. Install dependencies and start the Node.js Express server.
-4. Install dependencies and start the React server.
+1. Create an App Services App based on the `edge-server.todo` template app.
+2. Install Edge Server dependencies
```

Comment:
The install script handles prompting users to install missing dependencies (I think) so I think we can remove this step from the README.

Reply:
Nice! That's good DX. :)